src/js/components/molecules/main-navigation-item/main-navigation-item.ts

```diff
@@ -0,0 +1,121 @@
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import { MainNavigationItemProps } from './main-navigation-item.models';
+import WithRender from './main-navigation-item.vue.html';
+
+import BrowserCapabilities from 'utils/browser-capabilities';
+
+@WithRender
+@Component({
+    name: 'MainNavigationItem',
+})
+
+export class MainNavigationItem extends Vue implements MainNavigationItemProps {
+    @Prop({ type: Boolean, required: false }) mainNavOpen?: boolean;
```

Comment:
`required: false` can be removed; as far as i know, its the default value for `required`

Reply:
I know, just to be more clear